cmd/remove.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/wtsi-hgi/go-softpack-builder/remove"
+	"github.com/wtsi-hgi/go-softpack-builder/s3"
+)
+
+var removeCmd = &cobra.Command{
+	Use:   "remove",
+	Short: "Remove an environment",
+	Long: `Remove an environment.
+
+Remove an existing environment; it's entry in the Core artefacts repo, the
+module files and the singularity image and symlinks.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) < 1 {
+			die("require a environment path")
+		}
+
+		const maxArgs = 2
```

Comment:
I'd expect consts to be below the import. Also, it doesn't seem to be a "max" value, but an exact required value. numArgs?